.gitignore

```diff
@@ -8,3 +8,4 @@ user.bazelrc
 .vscode
 .cache
 docs/
+MODULE.bazel.lock
```

Comment:
FYI [best practices](https://bazel.build/external/lockfile#best-practices) seem to be to include the lockfile in VCS

Reply:
The lock file is a large file with a lot of hashes which is hard to verify. I suspect it makes sense if one uses a different repository than the bazel-central-repository, but it does not really serve a purpose otherwise: the BCR rules are so that a version number uniquely identifies a hash as they don't have a policy to switcaroo these later. 

So, I am not convinced that it brings anything to the table, though if there is compelling reason we can add it later.

Reply:
(as discussed offline: I am happy for some follow-up PR)